api/domain_handlers_test.go

```diff
@@ -24,3 +34,89 @@ func TestNormalizeDomain(t *testing.T) {
 		}
 	}
 }
+
+// TestIneligible tests that IneligibleDomainState Database is populated when the Ineligible endpoint is called.
+func TestIneligible(t *testing.T) {
+	api, _, mockHstspreload, mockPreloadlist := mockAPI(0 * time.Second)
+
+	// database.Scan values for testing
+
+	emptyScan := database.Scan{
+		Issues: []hstspreload.Issues{emptyIssues},
+	}
+
+	warningScan := database.Scan{
+		Issues: []hstspreload.Issues{issuesWithWarnings},
+	}
+
+	errorScan := database.Scan{
+		Issues: []hstspreload.Issues{issuesWithErrors},
+	}
+
+	TestEligibleResponses := map[string]hstspreload.Issues{
+		"garron.net":   emptyIssues,
+		"badssl.com":   issuesWithWarnings,
+		"chromium.org": emptyIssues,
+		"godoc.og":     issuesWithErrors,
+		"dev":          issuesWithWarnings,
+		"example.com":  issuesWithErrors,
```

Comment:
Instead of using real-world examples (or domain names that look close to real domains, e.g. `godoc.og`), this would be a good place to use RFC 2606 testing/documentation names (e.g. ending in `.test` or `.example`), and give them descriptive names. For example, when I see `garron.net` in this test file, I have to look at multiple places to understand what it's testing. If it's named `preloaded-bulk-18-weeks-no-issues.test`, then it's clearer what its purpose is throughout the test.